packages/react/src/components/Toggletip/index.tsx

```diff
@@ -331,19 +331,18 @@ interface ToggletipContentProps {
  * `children` passed in as a prop inside of `PopoverContent` so that they will
  * be rendered inside of the popover for this component.
  */
-export function ToggletipContent({
-  children,
-  className: customClassName,
-}: ToggletipContentProps) {
+export const ToggletipContent = React.forwardRef<
+  HTMLDivElement,
+  ToggletipContentProps
+>(function ToggletipContent({ children, className: customClassName }, ref) {
```

Comment:
The newly added `ref` isn't used anywhere in this component, so it will [always be `null`](https://react.dev/reference/react/forwardRef#my-component-is-wrapped-in-forwardref-but-the-ref-to-it-is-always-null)

Reply:
Thanks Taylor for the suggestion, Now passing the ref to the component and added a condition in parent element while passing ref to this one.